### PR TITLE
feat(service): use slurm `sinfo` command to improve "cluster load" indicator

### PR DESCRIPTION
**Pull request by MartinBelthle**
The branch name does not have anything to do with this PR, my bad.

There's a PR linked to it on antares-cd which includes the `partition` in the `application.yaml` file.
**This PR has been merged.**

### CODE REVIEW

**Comment by MartinBelthle** [comment]
This only works in production as the command is `--partition calin1`. It needs to works on every environment. 

Sylvain proposed to include the partition in the `application.yaml` and give the information to the launcher.sh script.

**Comment by MartinBelthle** [comment]
That's what i did so this PR is linked to this one on the launcher : https://github.com/AntaresSimulatorTeam/antares-launcher/pull/58 and this one on antares_cd : 

NB : All tests pass on AntaREST when combining with the PR on antares-launcher

**Review by laurent-laporte-pro — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -14,6 +14,7 @@ jsonref~=0.2
 MarkupSafe~=2.0.1
 numpy~=1.22.1
 pandas~=1.4.0
+paramiko~=2.12.0
```
Comment (laurent-laporte-pro) [inline requirements.txt:17]: Pourquoi a-t-on ajouté cette dépendance ?

**Comment by laurent-laporte-pro** [comment]
Curieusement, en intégration, l'indicateur donne :
```json
{
  "allocatedCpuRate": 0,
  "clusterLoadRate": 0.6666666666666667,
  "nbQueuedJobs": 2,
  "launcherStatus": "SUCCESS"
}
```

Alors qu'_a priori_, il n'y a rien qui tourne sur ce serveur…

**Review by laurent-laporte-pro — CHANGES_REQUESTED** [review]
Mise à jour de la documentation pour le nouveau paramètre `partition`.

Meilleure gestion du paramètre `partition` lorsqu'il est vide.

Gestion des erreurs lorsque la partition n'existe pas.

**Review by laurent-laporte-pro — APPROVED** [review]

**Pull request merged**